tests/ci/cdk/cdk/codebuild/github_ci_linux_x86_omnibus.yaml

```diff
@@ -12,7 +12,7 @@ batch:
         type: LINUX_CONTAINER
         privileged-mode: false
         compute-type: BUILD_GENERAL1_SMALL
-        image: ECR_REPO_PLACEHOLDER:ubuntu-20.04_gcc-8x_latest
+        image: ECR_REPO_PLACEHOLDER:ubuntu-20.04_clang-8x_latest
```

Comment:
The typo didn't have an effect on the build, or it does but was not noticed due to CI failure anyway?

Reply:
It did, but the CI was going to fail anyways